### DISCUSSION TITLE
Program not running when attemting to use SVG graphics

### DISCUSSION THREAD

**Discussion by AWake01** [andymeneely/squib#359] [Q&A]
![image](https://user-images.githubusercontent.com/40424954/145970886-fbe0c982-daca-46c0-a827-633e5bf65a42.png)
![image](https://user-images.githubusercontent.com/40424954/145971203-c6347dc1-93e3-45ef-a45a-507d80b5ea9b.png)
Hi, following the tutorials I am unable to get SVG images to appear. Am I missing some prerequisite gem?

**Comment by andymeneely** [comment]
Huh that doesn't look right.  Did `gem install squib` finish cleanly?

**Comment by neoaez** [comment]
I'm having this exact same issue. I've tried a fresh Ruby/squib install on 2 different computers with the exact same result. I created an issue for this. Would love to know how to get this working again.
